build.sh

```diff
@@ -2,4 +2,5 @@ xbuild Gaillard.SharpCover/Program.csproj \
 && xbuild Gaillard.SharpCover.Tests/ProgramTests.csproj \
 && mono Gaillard.SharpCover/bin/Debug/SharpCover.exe instrument travisCoverageConfig.json \
 && nunit-console Gaillard.SharpCover.Tests/bin/Debug/ProgramTests.dll \
-&& mono Gaillard.SharpCover/bin/Debug/SharpCover.exe check
+&& mono Gaillard.SharpCover/bin/Debug/SharpCover.exe check || true \
```

Comment:
this branch will need to be moved into master eventually or with this PR but for the build to still check this project correctly I think the check status should be saved and still used as the last exit status probably ANDed with the execution of view (basic and html).